src/components/TodoListItem.js

```diff
@@ -13,34 +11,36 @@ function TodoListItem({
     handleChange(todo.id);
   };
 
-  TodoListItem.propTypes = {
-    todo: PropTypes.object.isRequired,
-    onRemoveTodo: PropTypes.func.isRequired,
-    isChecked: PropTypes.bool.isRequired,
-    handleChange: PropTypes.func.isRequired,
-  };
-  
   return (
-    <ul>
-      <li>
-        <div className="item-container">
+    <li>
+      <div className={style.todoListThreeColumn}>
+        <div className={style.todoListColumn1}>
           <input
```

Comment:
I'm getting a console "issue"(never seen one of these before), about how this input needs a `name` or `id`

<img width="590" alt="Screen Shot 2023-10-02 at 4 41 06 PM" src="https://github.com/catbus00/react-todo/assets/12771651/9298a374-ca5c-45b7-81c6-750bcac38e56">

Reply:
This was due to not labeling my checkbox id, which has been updated with  id={`checkbox-${todo.id}`}. I've labeled it with a commit message